modules/removing-bare-metal-hosts-from-provisioner.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/managing-bare-metal-hosts.adoc
+
+:_content-type: PROCEDURE
+[id="removing-bare-metal-hosts-from-provisioner_{context}"]
+= Removing bare metal hosts from the provisioner node
+
+In certain circumstances it is desirable to temporarily remove bare metal hosts from the provisioner node.
```

Comment:
I might avoid a phrase like _it is desirable_ in tech docs. The IBM Sytle Guide says not to use _desire_ and use _want_ instead.
Maybe:  In certain circumstances, you might {want}/{need} to temporarily remove bare metal hosts....